src/index.js

```diff
@@ -1,12 +1,24 @@
 export default function (Vue, {
   name: defaultStoreName = 'persist:store',
+  expires: defaultExpires = 0,
   read = k => localStorage.getItem(k),
   write = (k, v) => localStorage.setItem(k, v)
 } = {}) {
   const cache = {}
+  const clear = k => localStorage.removeItem(k)
+  const isExpired = expires => expires && new Date().getTime() > expires
 
-  Vue.prototype.$persist = function (names, storeName = defaultStoreName) {
-    const data = cache[storeName] = JSON.parse(read(storeName) || '{}')
+  Vue.prototype.$persist = function (names, storeName = defaultStoreName, storeExpires = defaultExpires) {
+    const store = JSON.parse(read(storeName) || '{}')
```

Comment:
cache is used across multiple `$persist` calls for the same store, imagine `this.$persist(['foo'])` and `this.$persist(['bar'])`, they should use the same store instead of two local stores to keep in sync.